src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -376,18 +376,15 @@
                                      max-decimals  (if crypto-currency? crypto-decimals 2)
                                      regex-pattern (str "^\\d*\\.?\\d{0," max-decimals "}$")
                                      regex         (re-pattern regex-pattern)]
-                                 (when (and (not loading-routes?)
-                                            (re-matches regex new-text))
+                                 (when (re-matches regex new-text)
```

Comment:
To safetly implement this change, isn't enough just removing this condition.

We should also ignore/cancel/discard the pending request previously sent since it's not valid anymore.

Ideally, we would like:

1. User enters `0.001`
2. We ask for the routes and the input loads
3. While loading, user enters `0.002`
4. We ask for the NEW routes and the input loads
5. We receive the routes for `0.002` and the input stops loading

But with this implementation we will get:

1. User enters `0.001`
2. We ask for the routes and the input loads
3. While loading, user enters `0.002`
4. **Routes for `0.001` are received and stored in re-frame** (also maybe this makes the UI to blink and create a misleading db state)
5. We ask for the NEW routes and the input loads
6. We receive the routes for `0.002` and the input stops loading

So, what will happen if while executing `5` the request fails or takes too long to return? are the routes received in `4` interpreted as valid even if they come from a stale input value?

I think we should update the mechanism to ask for routes. Once an input has changed its value while the routes are being fetched, we should discard, cancel or ignore the request and response for those routes and ask for a new ones.

Reply:
> We should also ignore/cancel/discard the pending request previously sent since it's not valid anymore.

@ulisesmac I think we are already doing that. @briansztamfater can you please correct me if I am wrong, isn't [that line](https://github.com/status-im/status-mobile/blob/develop/src/status_im/contexts/wallet/send/events.cljs#L28) checking if the suggested route response in the right one using the timestamp?

> also maybe this makes the UI to blink
That blinking is another issue, it's from the bug introduced to the input component recently

Reply:
Thank you Omar, as long as this behavior is covered, everything is ok 👍

Reply:
> > We should also ignore/cancel/discard the pending request previously sent since it's not valid anymore.
> 
> @ulisesmac I think we are already doing that. @briansztamfater can you please correct me if I am wrong, isn't [that line](https://github.com/status-im/status-mobile/blob/develop/src/status_im/contexts/wallet/send/events.cljs#L28) checking if the suggested route response in the right one using the timestamp?

I think that should effectively cover this case. But anyway let's have @status-im/mobile-qa be aware of this case so it is properly tested suring manual QA 👍